spec/controllers/users_controller_spec.rb

```diff
@@ -23,22 +23,58 @@
 
   describe '#update' do
     it 'updates all attributes' do
-      user_params.merge! top_3_interests: ["Arts", "Music", "Crafting"]
+      subject = user_params.merge! top_3_interests: ["Arts", "Music", "Crafting"]
```

Comment:
It seems like you may have meant to make this assignment on line 28; rspec/route helpers dutifully convert this hash into a request and the test passes,but I am not sure it is what you meant.

This also passes, but doesn't make any sense:
```
expect(user_params).to redirect_to(user_path)
```

Please review whether this is the behavior you intended to test.